### DISCUSSION TITLE
Two datasets, rendering on same treemap - one rtl and other ltr

### DISCUSSION THREAD

**Discussion by PrivateRoDee** [kurkle/chartjs-chart-treemap#185] [Q&A] [answered]
Hello, I wanted to render a chart which looks something like this - 
![Screenshot 2023-05-09 163927](https://github.com/kurkle/chartjs-chart-treemap/assets/91798542/744dc10d-ca0b-45cb-a29d-a3ff6d2e252b)

I tried to go through the documentation but couldn't find properties that could be set to render something like the above screenshot. Is there a way to do this in treemap plugin?

**Comment by kurkle** [comment, marked as answer]
I don't think this is currently possible. Looks like a good use case and something that I think could be quite easily supported. I'll create an issue from this.
